test/complete-build-tests/test-reference.html

```diff
@@ -11,18 +11,24 @@
   <div id="mocha"></div>
 
   <!-- mocha -->
-  <script src="../node_modules/mocha/mocha.js" type="text/javascript" charset="utf-8"></script>
+  <script src="../../node_modules/mocha/mocha.js" type="text/javascript" charset="utf-8"></script>
   <script type="text/javascript" >
     // This will be overridden by mocha-helper if you run with grunt
     mocha.setup('tdd');
     mocha.reporter('html');
   </script>
 
   <!-- Include your assertion lib of choice -->
+  <script src="../js/chai.js" type="text/javascript" ></script>
+  <script src="../js/bind.js" type="text/javascript" ></script>
+  <script src="../js/sinon.js" type="text/javascript" ></script>
+  <script src="../js/modernizr.js"></script>
+
   <script src="js/chai.js" type="text/javascript" ></script>
   <script src="js/bind.js" type="text/javascript" ></script>
   <script src="js/sinon.js" type="text/javascript" ></script>
   <script src="js/modernizr.js"></script>
+
```

Comment:
looks like there are duplicate links here? I think the second set should be removed.